src/components/home/partners/SecondaryPartners.jsx

```diff
@@ -60,35 +60,36 @@ const LargeScreenComponent = ({
       }}
       key={company}
     >
-      <Grid item md={6} {...partnerGridStyle} order={logoOrder}>
+      <Grid item {...partnerGridStyle} md={5} order={logoOrder}>
         <a href={website} target="_blank" rel="noopener noreferrer">
           <Box
             component={'img'}
             alt={`${company} logo`}
             aria-label={`${company} logo`}
             src={partnerLogo}
-            mb={'150px'}
             width={'250px'}
           ></Box>
         </a>
       </Grid>
-      <Grid item pl={'6%'} {...partnerGridStyle} order={contentOrder}>
-        <Typography variant="body1" p={'5% 15% 3% 0'}>
+      <Grid item {...partnerGridStyle} md={7} order={contentOrder}>
+        <Typography variant="body1" p={'5% 10% 3% 10%'}>
```

Comment:
i think padding here can be `0% 10% 3% 10%`, also the `pt: '20px'` can go. 
```js 
const partnerGridStyle = {
  xs: 12,
  md: 6,
  display: 'flex',
  justifyContent: 'center',
  alignItems: 'center',
  flexDirection: 'column',
  pt: '20px' <----
};
```

Reply:
@Jared-Krajewski 

I pushed a commit to adjust the padding & alignment. I aligned items top, to be more like the figma file.

The CETI card looks a little off to me but not terrible there was no CETI in the initial figma mockup for comparison.

Note, I moved the 5% padding to the parent `grid` element. If I do the same padding on both child elements, they don't come out exactly equal. 🥳 I think this is fixable, but it seemed silly to waste too much time chasing it if padding the parent element works to make them the same size.

![Screen Shot 2024-02-06 at 5 04 16 PM](https://github.com/codeforpdx/codepdx_website/assets/3335277/87113f57-572f-4192-8474-c10e41d9b6ee)